functions-and-scope.js

```diff
@@ -53,13 +66,23 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade([6, 4, 5]) geeft xxxx
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
 
+function averageGrade(grades) {
+    let sum = 0;
+    for (const grade of grades) {
+        sum += grade;
+    }
+    return Math.round(sum / grades.length * 100) / 100;
```

Comment:
Goed hoor, je kan ook 
```suggestion
    return (sum / grades.length).toFixed(2);
```
doen